src/detection/cpu/cpu_linux.c

```diff
@@ -247,6 +248,9 @@ static const char* parseCpuInfo(
             #if !(__arm__ || __aarch64__)
             (cpu->name.length == 0 && ffParsePropLine(line, "model name :", &cpu->name)) ||
             (cpu->vendor.length == 0 && ffParsePropLine(line, "vendor_id :", &cpu->vendor)) ||
+            //Is it cheaper to just parse every physical id or to check if it's already set to the parsed value?
+            (cpuPhysicalId->length == 0 && ffParsePropLine(line, "physical id:", cpuPhysicalId)) ||
+            (cpuPhysicalId->length > 0 && ffParsePropLine(line, "physical id:", cpuPhysicalId)) ||
```

Comment:
This is weird. `cpuPhysicalId->length == 0 || cpuPhysicalId->length > 0` doesn't make sense to me.